pages/mesosphere/dcos/services/kafka/2.7.0-2.3.0/configuration/index.md

```diff
@@ -82,3 +82,63 @@ The grace period must also be respected when a broker is shut down before replac
 <p class="message--note"><strong>NOTE: </strong> Before performing this configuration change, you must create the necessary <a href="/mesosphere/dcos/latest/security/ent/secrets/">DC/OS Secrets</a> for the options: access_file, password_file, key_store, key_store_password_file, trust_store and trust_store_password_file.</p>
 
 Refer to [Secure JMX](/mesosphere/dcos/services/kafka/2.6.0-2.2.1/advanced/#secure-jmx-enterprise) for a more detailed configuration process.
+
+## Configuring Volume Profiles
+
+Volume profiles are used to classify volumes. For example, users can group SSDs into a “fast” profile and group HDDs into a “slow” profile. 
+
+<p class="message--note"><strong>NOTE: </strong>Volume profiles are immutable and therefore cannot contain references to specific devices, nodes or other ephemeral identifiers.</p> 
+
+[DC/OS Storage Service (DSS)](https://docs.d2iq.com/mesosphere/dcos/services/storage/1.0.0/) is a service that manages volumes, volume profiles, volume providers, and storage devices in a DC/OS cluster.
```

Comment:
Let's move this line above L88 because DSS is a the service that manages volume profiles. Then the next sentence would be describing what volume profiles are. As in..
```
DSS is a service that manages volumes, volume profiles, ...

Volume profiles are used to classify volumes ...
```